programs/programs/urgent_needs/co/test_first_step_savings.py

```
@@ -0,0 +1,271 @@
+        calculator = FirstStepSavings(self.screen, self.urgent_need, Dependencies(), {})
+
+        self.assertFalse(calculator.eligible())
+
+    def test_not_eligible_no_savings_interest(self):
+        """Test ineligibility when household hasn't expressed interest in savings"""
+        # Set needs_savings to False
+        self.screen.needs_savings = False
+        self.screen.save()
+
+        # Add an eligible child
+        child = HouseholdMember.objects.create(screen=self.screen, relationship="child", age=2)
```

Ruff (0.13.0)

Local variable `child` is assigned to but never used

Remove assignment to unused variable `child`

(F841)